mycli/main.py

```diff
@@ -105,7 +104,19 @@ def register_special_commands(self):
                 aliases=('\\r', ))
         special.register_special_command(self.refresh_dynamic_completions, 'rehash',
                 '\\#', 'Refresh auto-completions.', arg_type=NO_QUERY, aliases=('\\#',))
-
+        special.register_special_command(self.change_table_format, 'tableformat', 
+                '\\T', 'Change Table Type.', aliases=('\\T',)) 
```

Comment:
You need to add an extra parameter here called `case_sensitive=True`. Because as it stands right now this is overriding `\t` which turns on/off timing information. 

Also the third parameter is intended as a documentation string for the shortcut, so it should read `\\T [format_name]`.